google_guest_agent/main.go

```diff
@@ -270,12 +285,11 @@ func closer(c io.Closer) {
 
 func main() {
 	ctx := context.Background()
-	var action string
-	if len(os.Args) < 2 {
-		action = "run"
-	} else {
-		action = os.Args[1]
-	}
+	flag.StringVar(&action, "action", "run", "the action to run")
```

Comment:
Please don't add flags to this binary, it's not meant to be run on the cmdline and we have existing ways users are expected to configure it, either via metadata or config file